bff/test/Bff.Tests/Endpoints/Management/LoginEndpointTests.cs

```diff
@@ -28,6 +28,86 @@ public async Task login_should_allow_anonymous()
         response.StatusCode.ShouldNotBe(HttpStatusCode.Unauthorized);
     }
 
+    [Fact]
+    public async Task silent_login_should_challenge_and_redirect_to_root()
+    {
+        var response = await BffHost.BrowserClient.GetAsync(BffHost.Url("/bff/silent-login?redirectUri=/"));
+
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith("/bff/login");
+        response.Headers.Location!.ToString().ShouldContain("redirectUri=/");
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        response = await BffHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(IdentityServerHost.Url("/connect/authorize"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+
+        await IdentityServerHost.IssueSessionCookieAsync("alice");
+        response = await IdentityServerHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(BffHost.Url("/signin-oidc"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        response = await BffHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldBe("/bff/silent-login-callback");
+    }
+
+    [Fact]
+    public async Task can_also_issue_silent_login_with_prompt()
+    {
+        var response = await BffHost.BrowserClient.GetAsync(BffHost.Url("/bff/login?prompt=none"));
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(IdentityServerHost.Url("/connect/authorize"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        await IdentityServerHost.IssueSessionCookieAsync("alice");
+        response = await IdentityServerHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(BffHost.Url("/signin-oidc"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        response = await BffHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldBe("/bff/silent-login-callback");
+    }
+
+    [Fact]
+    public async Task login_with_unsupported_prompt_is_rejected()
+    {
+        var response = await BffHost.BrowserClient.GetAsync(BffHost.Url("/bff/login?prompt=select")); // select is not supported now. 
```

Comment:
select_account would be a better value to test with, as that is a value in the connect spec.

Reply:
Actually, select_account is supported and doesn't return an error. 

I've updated it to a clearly not supported prompt.